hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -500,7 +502,7 @@ private OzoneManager(OzoneConfiguration conf, StartupOption startupOption)
     this.isSecurityEnabled = OzoneSecurityUtil.isSecurityEnabled(conf);
     this.peerNodesMap = omhaNodeDetails.getPeerNodesMap();
     this.omNodeDetails = omhaNodeDetails.getLocalNodeDetails();
-
+    this.threadNamePrefix = omNodeDetails.threadNamePrefix();
```

Comment:
Let's keep the assignment in line 532, after `this.omNodeDetails.getNodeId() == null` (which might affect the prefix).